tests/s3/run.sh

```diff
@@ -28,7 +28,7 @@ DBPATH="$TEST_DIR/s3.mydump"
 export MINIO_ACCESS_KEY=s3accesskey
 export MINIO_SECRET_KEY=s3secretkey
 export MINIO_BROWSER=off
-export S3_ENDPOINT=127.0.0.1:9000
+export S3_ENDPOINT=127.0.0.1:9900
```

Comment:
Why change port?

Reply:
some CI keep failing because of port conflict (see this PR's history)